examples/example.py

```diff
@@ -28,112 +26,104 @@ def timeIntegration(context, steps, initialSteps):
+    alch_sim = app.Simulation(prmtop.topology, system,
+                              alch_integrator, platform)
+    alch_sim.context.setPeriodicBoxVectors(*inpcrd.boxVectors)
+    # Generate Alchemical System
+    factory = AbsoluteAlchemicalFactory(system, ligand_atoms,
+                                        annihilate_sterics=True,
+                                        annihilate_electrostatics=True)
+    alch_system = factory.createPerturbedSystem()
+
+    # Generate NC Integrator/Contexts
+    nc_integrator = ncmc_switching.NCMCVVAlchemicalIntegrator(temperature,
+                                               alch_system,
                                                functions,
                                                nsteps=nstepsNC,
                                                direction='insert',
                                                timestep=0.001*unit.picoseconds,
                                                steps_per_propagation=1)
 
-    nc_context = mm.Context(alchemical_system, nc_integrator, platform)
-    if inpcrd.boxVectors is not None:
-        nc_context.setPeriodicBoxVectors(*inpcrd.boxVectors)
+    nc_context = openmm.Context(alch_system, nc_integrator, platform)
+    nc_context.setPeriodicBoxVectors(*inpcrd.boxVectors)
+
+    # Initialize BLUES engine
+    blues_run = ncmc.SimNCMC(temperature, residueList=ligand_atoms)
 
     #during the ncmc move, perform a rotation around the center of mass at the start of step 49 (again to maintain symmetry of ncmc move
-    ncmove = [[practice_run.rotationalMove, [49]]]
+    rot_step = (nstepsNC/2) -1
+    nc_move = [[blues_run.rotationalMove, [rot_step]]]
 
     # actually run
-    practice_run.get_particle_masses(testsystem, residueList=lig_atoms)
-    practice_run.runSim(md_sim, nc_context, nc_integrator,
-                        dummy_sim, movekey=ncmove,
-                        niter=numIter, nstepsNC=nstepsNC, nstepsMD=nstepsMD,
-                        alchemical_correction=True)
+    blues_run.get_particle_masses(system, residueList=ligand_atoms)
+    blues_run.runSim(md_sim, nc_context, nc_integrator,
+                    alch_sim, movekey=nc_move,
+                    niter=numIter, nstepsNC=nstepsNC, nstepsMD=nstepsMD,
+                    alchemical_correction=True)
 
     nc_time = timeIntegration(nc_context, nstepsNC, 5)
-    md_time = timeIntegration(md_sim.context, nstepsMD, 20)
+    md_time = timeIntegration(md_sim.context, nstepsMD, 25)
 
     nc_steps = int(nstepsNC*1.0/nc_time)
     md_steps = int(nstepsMD*1.0/md_time)
 
     print('NC: Integrated %d steps in %g seconds' % (nstepsNC, nc_time))
     print('MD: Integrated %d steps in %g seconds' % (nstepsMD, md_time))
 
-    print('NC: %g ns/day' % (dt/2*nstepsNC/nc_time).value_in_unit(unit.nanoseconds))
-    print('MD: %g ns/day' % (dt*nstepsMD/nc_time).value_in_unit(unit.nanoseconds))
+    print('NC: %g ns/day' % (dt/2*nstepsNC*86400/nc_time).value_in_unit(unit.nanoseconds))
```

Comment:
The number 86400 here is not immediately obvious to me. Is this seconds in a day or some such?